### PR TITLE
website hebrew translation

### PR DESCRIPTION
**Pull request by kessler**
I actually finished most of it yesterday, but then I got called home and totally forgot to commit :)

I tried to use gender neutral language as much as possible. Also I couldn't think of a word in hebrew for checkbox so I made one up, "תיבת סימון" not very original but I think it gets the message delivered.

### CODE REVIEW

**Comment by RaMMicHaeL** [comment]
Thanks! That would probably require some adjustments to make the website look correctly for RTL, so I'll do the integration when I have some spare time.

File: website/he.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
+jquery_lang_js.prototype.lang.nl = {
```
Comment (RaMMicHaeL) [inline website/he.js]: `nl` -> `he`.

File: website/he.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+	
+	// Header
+	
+	'Home':
+		'בית',
+	'Press':
+		'לחץ',
```
Comment (RaMMicHaeL) [inline website/he.js]: That's a different kind of press :)
`עיתונות`, perhaps?

File: website/he.js [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+		'Bug/feature tracker',
+	'Blog':
+		'בלוג',
+	'Language':
+		'שפה',
+	
+	// Footer
+	
+	'Copyright © 2015 Reason Company Software Inc.':
+		'Copyright © 2015 Reason Company Software Inc. Nederlandse vertaling: <a href="mailto:contact@studio384.be">Studio 384</a>, S. Paternotte.',
```
Comment (RaMMicHaeL) [inline website/he.js]: Please update the translator information.

File: website/he.js [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+	
+	'Updates automatically':
+		'מתעדכן באופן עצמאי',
+	'Install and forget. Unchecky automatically updates whenever a new version is available, so you don\'t have to worry about running the latest version.':
+		'להתקין ולשכוח. אנצ׳קי מתעדכן באופן עצמאי בכל פעם שאנחנו מוציאים גרסא חדשה, כך שלא צריך לדאוג לעדכון אנצ׳קי לגרסתו האחרונה לבד',
+	
+	'Watch Unchecky in action':
+		'צפו באנצ׳קי בפעולה',
+	'<a href="press#quotes">See what people are saying about Unchecky</a> or <a href="press#videos">watch more videos</a>.':
+		'<a href="press#quotes">מה המשתמשים אומרים על אנצ׳קי?</a> of <a href="press#videos">צפו בקטעי וידאו נוספים</a>.',
```
Comment (RaMMicHaeL) [inline website/he.js]: `of` doesn't seem related.

File: website/he.js [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+	
+	// Press
+	
+	'People are talking about Unchecky':
+		'אנשים מדברים מדברים על אנצ׳קי',
+	'Unchecky was also featured on the following international websites:':
+		':אנצ׳קי גם מוזכר בכתבות ברחבי האינטרנט',
+	'on':
+		'כגון',
```
Comment (RaMMicHaeL) [inline website/he.js]: More like `דרך`. [The page](http://unchecky.com/press).

File: website/he.js (hunk 2 above)
Comment (kessler) [inline website/he.js]: Ohh I get it now :)

File: website/he.js (hunk 3 above)
Comment (kessler) [inline website/he.js]: what is this:

 Nederlandse vertaling: <a href="mailto:contact@studio384.be">Studio 384</a>, S. Paternotte.'
 ?

File: website/he.js (hunk 5 above)
Comment (kessler) [inline website/he.js]: do you mean כגון should be דרך ?

File: website/he.js (hunk 3 above)
Comment (RaMMicHaeL) [inline website/he.js]: It's the "translated by" note in the native language. In your case should be something like "תורגם על ידי קסלר", optionally with an email/homepage link.

File: website/he.js (hunk 5 above)
Comment (RaMMicHaeL) [inline website/he.js]: Yes, or whatever fits [here](http://i.imgur.com/5oETVYn.png).

**Comment by RaMMicHaeL** [comment]
Merged manually. Thanks!
